cinema/management/commands/initadmin.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from user.models import User
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args: Any, **options: dict[str]) -> None:
+        if User.objects.get_or_create():
```

Comment:
Maybe we need to pass some args in get_or_create?

Reply:
I think its better to use .exists in this case. Don't forget to add `not` at the beginning of the if clause.
So the logic should be following - if the admin user does not exist, we will create a new one